docs/protocol/src/crypto/decaf377.md

```diff
@@ -22,10 +22,26 @@ construction works the same way inside and outside of a circuit and imposes no
 costs for lightweight, software-only applications, making it a good choice for
 general-purpose applications.
 
+## Curve Parameters
+
+The cofactor-4 Edwards curve defined over the BLS12-377 scalar field has the
+following parameters:
+
+* Base field: Integers mod prime $q=8444461749428370424248824938781546531375899335154063827935233455917409239041$
+* Elliptic curve equation: $ax^2 + y^2 = 1 + dx^2y^2$ with $a=-1$ and $d=3021$
+* Curve order: $4r$ where $r=2111115437357092606062206234695386632838870926408408195193685246394721360383$
+
+We use a conventional generator basepoint selected to have a convenient hex encoding: 0x080000000000000000000000000000000
+0000000000000000000000000000000. In affine coordinates this generator point has coordinates:
```

Comment:
I think this will get mangled in markdown formatting, since it's split over two lines.  We could do a code block:
```
0x0800000000000000000000000000000000000000000000000000000000000000
```